src/util/live_performance.js

```diff
@@ -87,11 +100,8 @@ function getCountersPerResourceType(resourceTimers) {
 function getResourceCategory(entry: PerformanceResourceTiming): string {
     const url = entry.name.split('?')[0];
 
-    // Code may be hosted on various endpoints: CDN, self-hosted,
-    // from unpkg... so this check doesn't include mapbox HTTP URL
-    if (url.includes('mapbox-gl.js')) return 'javascript';
-    if (url.includes('mapbox-gl.css')) return 'css';
-
+    if (isMapboxHTTPCDNURL(url) && url.includes('mapbox-gl.js')) return 'javascript';
+    if (isMapboxHTTPCDNURL(url) && url.includes('mapbox-gl.css')) return 'css';
```

Comment:
Is that correct that we collect metrics only for the Mapbox assets?

Reply:
Yes, that's correct. It was initially set to look at Mapbox assets hosted on any endpoint (other CDN, unpkg or self-hosted) but with the introduction of caching detection, we have to ensure that TAO: * is enabled otherwise this leads to a misinformed request cache number and we can only ensure that on services we own.

Going forward this is ok as Mapbox services are the only actionable ones regarding performance and the ones we should focus on measuring. I wonder if another useful piece of information would be to retrieve the ratio of self-hosted vs requests pointing out to the CDN for these assets.